lib/videojs-playlist-ui.vertical.less

```diff
@@ -1,66 +1,68 @@
 /* The default color for the playlist menu background, almost black */
 @background-color: #1a1a1a;
-/* The color used to emphasize the currently playing video and for
-hover effects */
-@highlight-color: rgba(26, 26, 26, 0.5);
+/* The color used to emphasize the currently playing video and for effects */
+@highlight-color: #141a21;
 /* The primary foreground color */
 @main-color: #fff;
-/* The alternate foreground color */
-@secondary-color: #a6a6a6;
 
 /* Rules common to mouse and touch devices */
 .vjs-playlist {
   list-style-type: none;
-  overflow: auto;
+
+  overflow: scroll;
   position: relative;
 
   .vjs-playlist-item {
     cursor: pointer;
     overflow: hidden;
     position: relative;
+    margin-bottom: 5px;
   }
 
   .vjs-playlist-thumbnail {
-    background-color: @secondary-color;
     display: block;
-    float: left;
   }
-  .vjs-playlist-thumbnail::before {
-    display: block;
-    position: absolute;
-    background-color: @highlight-color;
-    text-align: center;
+
+  .vjs-playlist-thumbnail-placeholder {
+    height: 100px;
+    background: #303030;
   }
+
   img {
+    display: block;
     width: 100%;
-    height: 100%;
   }
 
   .vjs-playlist-duration {
     background-color: rgba(26, 26, 26, 0.8);
-    left: 0;
-    padding: 2px 8px;
+    top: .5rem;
+    left: .5rem;
+    padding: 2px 5px 3px;
+    margin-left: 2px;
     position: absolute;
-    top: 0;
   }
   .vjs-selected .vjs-playlist-duration {
     display: none;
   }
 
-  cite {
+  .vjs-playlist-name {
     display: block;
     font-style: normal;
+    max-height: 2.5em;
     overflow: hidden;
```

Comment:
It was removed because it doesn't work with the new structure. I'm hoping that since the length limit is so much longer we won't need it anymore. If a need to have it arises, we'll have to figure out how to do it.